v3/mokey/mokey12x2/mokey12x2.json

```diff
@@ -0,0 +1,125 @@
+{
+  "name": "mokey12x2",
+  "vendorId": "0x6653",
+  "productId": "0x012A",
+  "matrix": { "rows": 4,"cols": 5 },
+  "keycodes": ["qmk_lighting"], 
+  "menus": [
```

Comment:
https://github.com/qmk/qmk_firmware/tree/master/keyboards/mokey/mokey12x2

There is no RGB stuff here?

Reply:
Mokey12x2.json is not being uploaded, but I don't know how to close it. Please help me close it. Thank you

Reply:
I cannot remove it for you. You must do it yourself